chipy_org/settings.py

```diff
@@ -88,16 +87,34 @@ def env_list(key, defaults=[], delimiter=','):
     "django.contrib.staticfiles.finders.AppDirectoriesFinder",
 ]
 
+USE_S3 = env_var("USE_S3", True)
+
+if USE_S3:
+    AWS_ACCESS_KEY_ID = os.environ.get('AWS_ACCESS_KEY_ID')
+    AWS_SECRET_ACCESS_KEY = os.environ.get('AWS_SECRET_ACCESS_KEY')
+    AWS_STORAGE_BUCKET_NAME = os.environ.get('AWS_STORAGE_BUCKET_NAME')
+    AWS_QUERYSTRING_AUTH = False
+    AWS_HEADERS = {
+        'Cache-Control': 'max-age=86400',
+    }
+    DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
+    #STATICFILES_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
```

Comment:
If we want to host staticfiles on S3, we can uncomment that.  I think we are OK hosting the staticfiles on Heroku since that's all stuff that's generated by the codebase and S3, and my understanding is that Heroku runs a collectstatic on each provision.